README.md

```diff
@@ -40,6 +40,16 @@ Check out live demo: [https://flotiq-nextjs-shop-2.netlify.app](https://flotiq-n
     FLOTIQ_API_KEY=YOUR FLOTIQ API KEY
     SNIPCART_API_KEY=YOUR SNIPCART API KEY
     ```
+   To add `SNIPCART API KEY` to your starter you need to log in to the website: https://snipcart.com/
+
+   After this you need to get into your account options and choose `API KEYS` line. 
+   To see how snipcart shop is working you need to copy `PUBLIC TEST API KEY` and paste it into your `.env` file instead `YOUR SNIPCART API KEY`.
+
+   You need to remember that this is just for test look, to get full working snipcart shop for commercial use you need to buy it on the website.
```

Comment:
I think there is a typo, it should be ` fully working ` instead of ` full working `